custom/icds_reports/views.py

```diff
@@ -2220,7 +2220,7 @@ def get(self, request, *args, **kwargs):
             return HttpResponse(error_message, status=400)
 
         query_filters = {'aggregation_level': AggregationLevels.AWC,
-                         # 'num_launched_awcs': 1,
+                         'num_launched_awcs': 1,
```

Comment:
Should this be `num_launched_awcs__gte`?

Reply:
hey not for this API as this only queries on agg_level=5(awc level) on which num_launched_awcs value could be 0/1